src/list-names.py

```diff
@@ -1,17 +1,14 @@
-import argparse
-
-
-def main():
-    argparser = argparse.ArgumentParser(
-        description="Extract the names from a simple-fastq file")
-    argparser.add_argument(
-        "fastq",
-        type=argparse.FileType('r')
-    )
-    args = argparser.parse_args()
-
-    print(f"Now I need to process the records in {args.fastq}")
-
-
-if __name__ == '__main__':
-    main()
+import argparse
+import sys
+
+parser = argparse.ArgumentParser(prog = "list_names")
+parser.add_argument("-path", nargs = 1, type = str, help = "Path to input fastq file")

```

Comment:
Breaking the program interface again, just as with the other two exercises. This ensures that your tool cannot pass the tests.